vocabularies/Offline.xml

```diff
@@ -49,7 +49,21 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
            attempt to upload any changes that are made to them locally.
            The service as implemented on the server is described by the CSDL document if
            all elements tagged as `Offline.ClientOnly` are removed together with all subsequently dangling references.</String>
-        </Annotation>        
+        </Annotation>
+      </Term>
+
+      <Term Name="LocalDraft" Type="Edm.String" Nullable="false" AppliesTo="EntitySet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Marks a local draft version of an entity set, which uses the same entity type as a non-draft entity set defined in the backend defined metadata."/>
+         <Annotation Term="Core.LongDescription">
+           <String>Typically for each entity type there is one corresponding entity set, however OData permits having multiple entity sets sharing the same entity type.
+            This can be taken advantage of to enable a facility for local (offline) drafts by defining an additional (draft) entity set for an existing entity type.
+            For example, with existing entity type "Order" and existing entity set "Orders", an additional "DraftOrders" entity set could be defined, annotated with
+            this annotation. When a client application creates an entity instance, it could be created in the "DraftOrders" entity set. A subsequent offline entity
+            upload operation will not upload any local draft entities.
+            Draft entities and their children need to be transitioned into non-draft entities to become uploadable.
+            The linked non-draft entity set using the same entity type can be defined in the string value, and the implicit default value of null indicates that the draft set is linked with the only other non-draft entity set using the same entity type.</String>
+        </Annotation>
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="Marks a local draft version of an entity set, which uses the same entity type as a non-draft entity set defined in the backend defined metadata."/>
         <Annotation Term="Core.LongDescription">
           <String>Typically for each entity type there is one corresponding entity set, however OData permits having multiple entity sets sharing the same entity type.
            This can be taken advantage of to enable a facility for local (offline) drafts by defining an additional local draft entity set for an existing server-side entity set and entity type.
            For example, with a server-side entity set `Orders` and entity type `Order`, an additional `DraftOrders` entity set could be defined, annotated with this annotation. 
            The annotation value is the name of the non-draft entity set using the same entity type.
            When a client application creates an entity instance, it could be created locally in the `DraftOrders` entity set. 
            A subsequent offline entity upload operation will not upload any local draft entities.
            Draft entities and their children need to be transitioned into non-draft entities to become uploadable.</String>
        </Annotation>
```

Reply:
That's my understanding: local draft entity sets are a specific flavor of ClientOnly.